src/test/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerTerminationValidatorTest.java

```diff
@@ -185,4 +188,34 @@ void validateTerminationDateAfterIncorporationDateWhenSameDay() {
                 .as("An error should not be produced when resignation date is the creation date")
                 .isEmpty();
     }
+
+    @Test
+    void validateSubmissionInformationInDateWhenValid() {
+        when(companyAppointment.getEtag()).thenReturn(ETAG);
+        final var officerFilingDto = OfficerFilingDto.builder()
+            .referenceEtag("etag")
+            .referenceAppointmentId(FILING_ID)
+            .resignedOn(LocalDate.of(2023, Month.JANUARY, 5))
+            .build();
+        officerTerminationValidator.validateSubmissionInformationInDate(request, officerFilingDto, companyAppointment, apiErrorsList);
+        assertThat(apiErrorsList)
+            .as("The Officers information is out of date. Please start the process again and make a new submission")
```

Comment:
This should be a comment that explains the error as it will be output if the test fails.